### ISSUE TITLE
Doesn't Auto create queue.js file in config directory

### ISSUE THREAD

**Issue by nasiriyima** [stitchng/adonis-queue#7]
After installation using 
`adonis install adonisjs-queue`

The package doesn't create `queue.js ` file in the config directory. So also documentation doesn't provide information on how to create, setup or configure the queue.js config file

**Comment by harlancleiton** [comment]
Copy the file located at node_modules/adonisjs-queue/config/queue.js for your project

Not a solution but it works

**Comment by stitchng** [comment]
Hello, @nasiriyima 

what operating system do you use ?

 what version of NodeJS / NPM do you use ?

**Comment by nasiriyima** [comment]
@stitchng 
Sorry I didn't give those specifics, find it below:
OS: Windows 10
NodeJS: v12.13.0
NPM: 6.12.0

By the way, I just wanted to find out if the package is compatible with other databases such as mysql. Although using RAM cached DBs are OK but sometimes, there maybe issues with deployment on entry tier cloud servers (with low RAM) on services such as digital ocean and AWS micro. Like currently I'm already having issues using the geoip-lite npm packge, because it uses a lot of RAM to cache files.

**Comment by nasiriyima** [comment]
> Copy the file located at node_modules/adonisjs-queue/config/queue.js for your project
> 
> Not a solution but it works

Yeah done that already, but just wanted to let the author know in case it was an error on their part. Thanks any ways!

**Comment by stitchng** [comment]
This issue is still being investigated. We will get back on this by next week.
